adb/adb_commands.py

```diff
@@ -194,19 +258,27 @@ def Pull(self, device_filename, dest_file='', progress_callback=None, timeout_ms
       timeout_ms: Expected timeout for any part of the pull.
```

Comment:
Can you document `progress_callback`? Or put a `TODO(fahhem)` for me to :)

Reply:
Done